services/core-api/app/api/projects/information_requirements_table/models/information_requirements_table.py

```diff
@@ -92,6 +97,51 @@ def send_irt_submit_email(self):
         body += f'<p>View IRT in Core: <a href="{link}" target="_blank">{link}</a></p>'
         EmailService.send_email(subject, recipients, body)
 
+    def send_irt_status_notifications(self, project_guid: str, status_code: str):
+        project: Project = Project.find_by_project_guid(project_guid)
```

Comment:
should be able to do one less query with self.project to get the project

Reply:
Tried that, and it wasn't returning with the full project including the project lead.